Makefile

```diff
@@ -56,3 +56,16 @@ deps:
 .PHONY: test
 test:
 	go test ./...
+	
+.PHONY: license-check
+license-check:
+	@echo "--> Checking license header..."
+	@licRes=$$(for file in $$(find . -type f -regex '.*\.sh\|.*\.go\|.*Docker.*\|.*\Makefile*' ! -path './vendor/*' ! -path './hack/tools.go' ) ; do \
```

Comment:
@AJEETRAI707 we can now remove the `! -path './hack/tools.go` , similar to other PR we make it work.

Reply:
@prateekpandey14  Yes sure. I will remove `! -path './hack/tools.go` this.